internal/pkg/cli/app_delete.go

```diff
@@ -293,9 +293,9 @@ func (o *deleteAppOpts) deleteAppConfigs() error {
 
 func (o *deleteAppOpts) deleteWs() error {
 	o.spinner.Start(deleteAppWsStartMsg)
-	if err := o.ws.DeleteAll(); err != nil {
-		o.spinner.Stop(log.Serror("Error deleting local workspace folder."))
-		return fmt.Errorf("delete workspace: %w", err)
+	if err := o.ws.DeleteWorkspaceFile(); err != nil {
+		o.spinner.Stop(log.Serror("Error deleting .workspace file."))
```

Comment:
Nit: It may be useful to interpolate the constant "workspace.workspaceSummaryFileName" here instead of hardcoding '.workspace', in case the name ever changes.

Reply:
I do like this idea! Issue is that I would need to make workspaceSummaryFileName global. Are we okay with this unless there is a another way I am not finding.

Reply:
that's right. you'll need to export it but i think it should be fine to do that.

Reply:
I'm okay with that since it's a constant and not a variable!

Reply:
Make senses 👍